index.js

```diff
@@ -64,25 +47,7 @@ module.exports = function ({ content, filename, sourceMap }) {
       + transpile(`${exportsTarget}.staticRenderFns = [${staticRenderFns}];`)
       + `\n${exportsTarget}.render._withStripped = true;`;
   }
-  const result = `${scriptPart}${compiledTemplate}`;
```

Comment:
The parseComponent function adds padding where the template/style tags were, so the script is still on the same line as the original vue file. This means you don't need source maps, which cuts down on overhead a little!